src/index.d.ts

```diff
@@ -224,4 +224,4 @@ declare const RNDateTimePicker: FC<
 >;
 
 export default RNDateTimePicker;
-export const DateTimePickerAndroid: typeof DateTimePickerAndroidType;
+export type DateTimePickerAndroid = typeof DateTimePickerAndroidType;
```

Comment:
btw I think this file should not be necessary at all now, its content will be replaced by types.ts and by other types across the code base